.changeset/fifty-rules-count.md

```diff
@@ -0,0 +1,13 @@
+---
+"@salt-ds/core": minor
+---
+
+Added `InteractableCardGroup` component to support selectable cards.
```

Comment:
```suggestion
Added `InteractableCardGroup` component to support selectable cards in a group.
```
or may be worth mentioning the support added vs using the cards in just a regular container or layout